utilities/decrypt_data/decrypt.py

```
@@ -0,0 +1,90 @@
+                    WHERE {CONFIG['id_column']} = %s
+                """
+                values = list(decrypted_data.values()) + [row_id]
+                cur.execute(update_query, values)
+                print(f"Updated ID {row_id}: {decrypted_data}")
+
+        conn.commit()
+        cur.close()
+        conn.close()
+        print("Decryption and update completed.")
+
+    except Exception as db_err:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)